app/assets/javascripts/short-form/templates/d6-status-vouchers.html.slim

```diff
@@ -8,7 +8,7 @@ form novalidate="" name="form.applicationForm" ng-submit="submitForm({path: 'dah
     .radio-group.padding-top--half role="radiogroup" aria-label="{{'D6_STATUS_VOUCHERS.TITLE' | translate}}"
       radio-block-item value='Yes' user='application' name='householdVouchersSubsidies' ng-required='true'
         | {{'T.YES' | translate}}
-      radio-block-item value='No' user='application' name='householdVouchersSubsidies' ng-required='true'
+      radio-block-item value='No' user='application' name='householdVouchersSubsidies' ng-required='true' ng-click="invalidateIncomeForm()"
```

Comment:
@kaptron does this ng-click essentially re-marks the section as incomplete/not valid so that it can run the validation check again?